Makefile

```diff
@@ -243,7 +243,7 @@ install-deps-for-semgrep-core: semgrep.opam
 	&& ./configure \
 	&& ./scripts/install-tree-sitter-lib
 	# Install OCaml dependencies (globally) from *.opam files.
-	opam install -y --deps-only ./ ./libs/ocaml-tree-sitter-core
+	LIBRARY_PATH="/opt/homebrew/lib" opam install -y --deps-only ./ ./libs/ocaml-tree-sitter-core
```

Comment:
Not a fan of this change here. Those targets are platform-agnostic and should not contain
homebrew specific stuff. For this you can use the homebrew-setup: target